Makefile

```diff
@@ -4,12 +4,14 @@ export DEVELOPMENT?=false
 export AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 export AWS_REGION?=us-west-2
 export IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+export KOPS_CLUSTER_NAME=$(shell cat /dev/urandom | LC_ALL=C tr -dc '[:alpha:]' | tr '[:upper:]' '[:lower:]' | head -c 32).prod.test.pdx.kops-ci.model-rocket.aws.dev
+export TEST_ROLE_ARN=arn:aws:iam::$(AWS_ACCOUNT_ID):role/$(TEST_ROLE_NAME)
```

Comment:
This account id will not be correct then right? It would use build account rather than test-build account